pkg/framework/host.go

```diff
@@ -413,13 +416,17 @@ func (h *Host) crd(crdName string) func() error {
 }
 
 func (h *Host) installVault() error {
+	if h.vaultToken == "" {
+		return microerror.Mask(missingVaultTokenError)
+	}
```

Comment:
IMO this should be done in the constructor. See https://github.com/giantswarm/e2e-harness/pull/95.